rueidisprob/bloomfilter.go

```diff
@@ -170,6 +212,32 @@ func NewBloomFilter(
 	}
 	hashIterations := numberOfBloomFilterHashFunctions(size, expectedNumberOfItems)
 
+	options := &BloomFilterOptions{}
+	for _, opt := range opts {
+		opt(options)
+	}
+
+	var existsMultiScript *rueidis.Lua
+	if options.enableReadOperation {
+		resp := client.Do(context.Background(), client.B().Info().Section("server").Build())
+		if resp.Error() != nil {
+			return nil, resp.Error()
+		}
+
+		info, err := resp.ToString()
+		if err != nil {
+			return nil, err
+		}
+
+		if parseRedisMajorVersion(info) < 7 {
```

Comment:
Users may run a redis cluster with mixed versions and a single INFO request is not enough for checking the version of each node. Actually, I think this check is unnecessary. Stating the version requirement in the comment of WithEnableReadOperation is enough.

Reply:
> Users may run a redis cluster with mixed versions and a single INFO request is not enough for checking the version of each node. 

Oh, i missed. thank you.

[8832748](https://github.com/redis/rueidis/pull/598/commits/8832748a2a85c64cc0e3e2b102d559fd11484710)